src/wp-includes/class-wp-scripts.php

```diff
@@ -489,19 +458,14 @@ public function add_inline_script( $handle, $data, $position = 'after', $standal
 		$script   = (array) $this->get_data( $handle, $position );
 		$script[] = $data;
 
-		// Maintain a list of standalone and non-standalone before/after scripts.
-		$standalone_key      = $standalone ? $position . '-standalone' : $position . '-non-standalone';
-		$standalone_script   = (array) $this->get_data( $handle, $standalone_key );
-		$standalone_script[] = $data;
-		$this->add_data( $handle, $standalone_key, $standalone_script );
-
 		return $this->add_data( $handle, $position, $script );
 	}
 
 	/**
 	 * Prints inline scripts registered for a specific handle.
 	 *
 	 * @since 4.5.0
+	 * @deprecated 6.3.0 Use methods get_inline_script_tag() or get_inline_script_data() instead.
 	 *
 	 * @param string $handle   Name of the script to add the inline script to.
 	 *                         Must be lowercase.
```

Comment:
```suggestion
	 * @param string $handle   Name of the script to print inline scripts for.
	 *                         Must be lowercase.
```
@westonruter @joemcgill I'm thinking this doc block property description is actually incorrect ([even in current core](https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/class-wp-scripts.php#L457-L471)). 

This is not the name of the script to add an inline script to, but rather the name of a script to _print inline scripts for_, not so?

Reply:
Yeah, looks like you're right. However, since this method is being deprecated here, maybe just leave it?

Reply:
@westonruter ya, I don't have a strict preference here, I'm fine with either. Might be best to leave it.

Reply:
+1 This was likely copy/pasta from the `add_inline_script` method when this was introduced. I think it would be nice to correct it, even as it's being deprecated, since it may remain in the codebase for a while even after deprecation 😉

Reply:
Suggestion applied in https://github.com/10up/wordpress-develop/pull/67/commits/6ae814d20affb212d02ccaadab5d8395d22e19c7